### ISSUE TITLE
(FS) Cache Server API?

### ISSUE THREAD

**Issue by jvizcar** [dgutman/webix_image_organizer#379]
Bug description: I was running the faceted search app using DSA instance 1. I wanted to switch to using DSA instance 2. I modified the config file by replacing the URL to DSA instance 2, deleted the container and volumes, and rebuild the app. When I used the application however the old DSA API was being used. The only way to solve this was to add more than one DSA instance on the config file, rebuild the app, and using the admin panel switch the servers.

It looks like the DSA API is being cached somehow, as it was being used even when it was no longer on the config file.

**Comment by jvizcar** [comment]
I am still getting this bug. I tried to build the image with no cache, then run it again, logout and log in, clear the database, and still got stuck showing collections for the first API I used. Adding two and switching between them finally fixed this. Not sure what the cause is. We are not going to support having multiple API / hosts, but this could still be a problem if the user switches API URLs. It is a niche problem, I would keep it in mind and work on it when time allows.
